src/Simplic.OxS.Data/NoSql/IOrganizationRepository.cs

```diff
@@ -17,5 +19,11 @@ public interface IOrganizationRepository<TId, TDocument, TFilter> : IRepository<
         /// <param name="queryAllOrganizations"></param>
         /// <returns>Entity</returns>
         Task<TDocument> GetAsync(TId id, bool queryAllOrganizations = false);
-    }
+
+        /// <summary>
+        /// Gets the collection of an document as an executeable.
+        /// </summary>
+        /// <returns></returns>
+		Task<IExecutable<TDocument>> GetCollection();
```

Comment:
Does this line really need a reference to: `HotChocolate`?

Reply:
Yes